lib/Frontend/Frontend.cpp

```diff
@@ -211,6 +211,7 @@ bool CompilerInstance::setUpASTContextIfNeeded() {
   // and single file builds.
   Invocation.getLangOptions().RecordRequestReferences
     = !isWholeModuleCompilation();
+  Invocation.getLangOptions().ModuleAliasMap = Invocation.getFrontendOptions().ModuleAliasMap;
```

Comment:
It needs to be in LangOpts so it can be looked up in name lookup / sema. I thought about converting to `Identifier` but it's defined in AST while LangOpts is in swift/basic.

Reply:
Could the one in `LangOptions` be the only one, then? Or could we copy it from `FrontendOptions` to somewhere in the AST (maybe the main module's ModuleDecl?).

Reply:
Will move to ModuleDecl, in a separate pr.